app/components/CreateWalletForm/ConfirmSeed/SeedEntry.js

```diff
@@ -91,7 +103,14 @@ class SeedEntry extends React.Component {
       break;
     }
   }
-
 }
 
+SeedEntry.prototypes = {
+  seedType: PropTypes.string,
+};
+
+SeedEntry.defaultProps = {
+  seedType: "words",
```

Comment:
Since you're using the "words" and "hex" constants (the seed type) in multiple files, I'd define them as constants (`SEED_TYPE_WORDS = "...", SEED_TYPE_HEX = ...`) somwhere and import them.